### ISSUE TITLE
Audioengine d3 fix breaks everything else

### ISSUE THREAD

**Issue by yoyoyonono** [yoyoyonono/nixos-config-file#1]
```

    wireplumber.configPackages = [
      (pkgs.writeTextDir "share/wireplumber/main.lua.d/50-alsa-config.lua" ''
        table.insert (alsa_monitor.rules, {
          matches = {
            { { "device.name", "matches", "alsa_card.*" }, },
          },
          apply_properties = {
            ["api.alsa.ignore-dB"] = true, 
          },
        })
      '')
    ];
```

fixes the problem but then all other audio stops working

**Comment by yoyoyonono** [comment]
almost fixed by dbb42eba9cfb30c650209bd2ba62ce1d69a9d7ee

**Comment by yoyoyonono** [comment]
looks like that fixed it actually
